modules/ossm-rn-deprecated-features.adoc

```diff
@@ -15,6 +15,25 @@ Deprecated functionality is still included in {product-title} and continues to b
 
 Removed functionality no longer exists in the product.
 
+== Deprecated and removed features {SMProductName} 2.4
+
+Support for `ServiceMeshControlPlane` v2.1 has been removed from the operators. Customers should upgrade their mesh deployments to use a later version of the `ServiceMeshControlPlane` resource.
```

Comment:
Actually, it was not removed, but is not supported.

Reply:
@jewertow thanks for pointing that out. How about something like this:

The v2.1 `ServiceMeshControlPlane` resource is no longer supported. Customers should upgrade their mesh deployments to use a later version of the resource.